### PR TITLE
Fix OpenAPI definitions for SafeState entity

### PR DESCRIPTION
**Pull request by hectorgomezv**
- Fixes some incomplete OpenAPI definitions for `SafeState` entity.

### CODE REVIEW

File: src/routes/safes/entities/safe-info.entity.ts [hunk 1]
```diff
@@ -20,11 +20,11 @@ export class SafeState {
   readonly owners: AddressInfo[];
   @ApiProperty()
   readonly implementation: AddressInfo;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: AddressInfo, nullable: true })
   readonly modules: AddressInfo[] | null;
```
Comment (fmrsabino) [inline src/routes/safes/entities/safe-info.entity.ts:24]: The type of this one should be an `Array<AddressInfo>`

Comment (hectorgomezv) [reply src/routes/safes/entities/safe-info.entity.ts]: True 🤦🏻‍♂️
Fixed in [b25cf00](https://github.com/safe-global/safe-client-gateway/pull/765/commits/b25cf0019e537d984a8e1bdabaa4b0144d362acb), thanks!

File: src/routes/safes/entities/safe-info.entity.ts [hunk 2]
```diff
@@ -20,11 +20,11 @@ export class SafeState {
   readonly owners: AddressInfo[];
   @ApiProperty()
   readonly implementation: AddressInfo;
-  @ApiPropertyOptional()
+  @ApiPropertyOptional({ type: Array<AddressInfo>, nullable: true })
```
Comment (fmrsabino) [inline src/routes/safes/entities/safe-info.entity.ts:23]: This won't work: you'd need to set it as `type: [AddressInfo]` or with `type: AddressInfo, isArray: true`

**Review by fmrsabino — CHANGES_REQUESTED** [review]

File: src/routes/safes/entities/safe-info.entity.ts (hunk 2 above)
Comment (hectorgomezv) [reply src/routes/safes/entities/safe-info.entity.ts]: Fixed in [1b99b95](https://github.com/safe-global/safe-client-gateway/pull/765/commits/1b99b95ba4cc5b36f637daf9fdec1f88afb15e68), had to rebase to include https://github.com/safe-global/safe-client-gateway/pull/766 and test it. Sorry about the back and forth 🙂

**Review by fmrsabino — APPROVED** [review]

**Pull request merged**